lms/djangoapps/certificates/utils.py

```diff
@@ -57,7 +57,7 @@ def emit_certificate_event(event_name, user, course_id, course_overview=None, ev
     data = {
         'user_id': user.id,
         'course_id': str(course_id),
-        'certificate_url': get_certificate_url(user.id, course_id, uuid=event_data['certificate_id'])
+        'certificate_url': get_certificate_url(course_id, uuid=event_data['certificate_id'])
```

Comment:
I'm curious why we still need `get_certificate_url` function, shouldn't it be removed as well?

Reply:
`get_certificate_url` function used to get based certificate URL for HTML representation of certificate